PhysicsPackages/eigenPhysicsPackage_class.f90

```diff
@@ -3,10 +3,18 @@ module eigenPhysicsPackage_class
   use numPrecision
   use universalVariables
   use endfConstants
-  use genericProcedures,              only : fatalError, printFishLineR, numToChar, rotateVector
+  use genericProcedures,              only : numToChar, rotateVector
+  use display_func,                   only : printFishLineR, statusMsg, printSectionStart, &
+                                             printSectionEnd, printSeparatorLine
+  use mpi_func,                       only : isMPIMaster, getWorkshare, getOffset, getMPIRank
+#ifdef MPI
+  use mpi_func,                       only : MASTER_RANK, MPI_Bcast, MPI_INT, MPI_COMM_WORLD, &
```

Comment:
Are we consistent with how we write MPI commands? Should they all be all caps or not? If they should then we should have 
```suggestion
  use mpi_func,                       only : MASTER_RANK, MPI_BCAST, MPI_INT, MPI_COMM_WORLD, &
```